grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -93,28 +93,21 @@ private void fetchRepositories() {
                         ++mPage,
                         PER_PAGE
                 )
-                .subscribe(new Observer<Repositories>() {
-                    @Override
-                    public void onCompleted() {
-                        Log.d(TAG, "onCompleted");
-                    }
-
-                    @Override
-                    public void onError(Throwable e) {
-                        Log.d(TAG, "onError");
-                    }
-
-                    @Override
-                    public void onNext(Repositories repositories) {
-                        Log.d(TAG, "onNext");
-                        mProgress.setVisibility(View.GONE);
-
-                        mAdapter.add(repositories.items);
-                        mAdapter.setIsLoading(false);
-                        isLoading = false;
-                        mTotal = mAdapter.getItemCount();
-                    }
-                });
+                .subscribe(
+                        repositories -> {
```

Comment:
good 👍 

just for fun can you make it like this?
`.subscribe(this::handleResponse, this::handleError, this::handleComplete)`